src/main/java/com/webdev/cosmo/cosmobackend/posts/dto/PostModel.java

```diff
@@ -0,0 +1,12 @@
+package com.webdev.cosmo.cosmobackend.posts.dto;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@Data
+public class PostModel {
+    private String title;
+    private String description;
+    private Long imageId;
```

Comment:
String imageId
Also add id to Post to component section, we want to return it in response ALWAYS